### PR TITLE
[cmake] Add cmake parameter to BuildSetup.bat

### PR DESCRIPTION
**Pull request by fetzerch**
First attempt to integrate cmake into the BuildSetup.bat script. This is not perfect as the packaging only takes the main executable from the cmake build and nothing more. Still good enough for the beginning to verify the build on Jenkins.

Would be happy to get some reviews from people that know batch scripting.
@paxxi, @Montellese, @ace20022 maybe

### CODE REVIEW

**Comment by MartijnKaijser** [comment]
Needs updating now we have Vs2015?

**Comment by fetzerch** [comment]
Yep, seen it too late. Done.

File: project/Win32BuildSetup/BuildSetup.bat [hunk 1]
```diff
@@ -157,6 +161,41 @@ set WORKSPACE=%CD%\..\..
...
+  ECHO ------------------------------------------------------------
+  ECHO Compiling %APP_NAME% branch %BRANCH%...
+
+  IF %buildmode%==clean (
+    RMDIR /S /Q %WORKSPACE%\kodi-build
+  )
+  MKDIR %WORKSPACE%\kodi-build
+  PUSHD %WORKSPACE%\kodi-build
+
+  call "C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\vcvarsall.bat" x86
```
Comment (Paxxi) [inline project/Win32BuildSetup/BuildSetup.bat]: don't think this is needed, cmake takes care of finding the build tools itself. If it must be here, should probably use %VS140COMNTOOLS%\..\..\VC\vcvarsall.bat x86

Comment (fetzerch) [inline project/Win32BuildSetup/BuildSetup.bat]: True, it's not needed, works well with the `-G "Visual Studio 14"` parameter alone.

**Comment by fetzerch** [comment]
Successful build http://jenkins.kodi.tv/job/WIN-32/8481/ (with unrelated addon failure).

**Pull request merged**